msticpy/data/drivers/local_osquery_driver.py

```diff
@@ -0,0 +1,231 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""Local Osquery Data Driver class - osquery.{results,snapshots}.log."""
+from pathlib import Path
+from typing import Union, Any, Dict, Optional, List
+
+import os
+import json
+import pandas as pd
+
+from .driver_base import DriverBase, QuerySource
+from ...common.pkg_config import settings
+from ...common.utility import export
+from ..._version import VERSION
+
+__version__ = VERSION
+__author__ = "juju4"
+
+
+@export
+class LocalOsqueryDriver(DriverBase):
```

Comment:
I did for velociraptor as just the read_json but for osquery, as there is extra processing, I decided to keep separate to avoid bloating LocalData driver.
Not sure if cleaner option, something like overloading class maybe

Reply:
Agreed